shared/build.gradle.kts

```diff
@@ -0,0 +1,31 @@
+plugins {
+	alias(libs.plugins.kotlin.multiplatform)
+	alias(libs.plugins.android.library)
+}
+
+kotlin {
+	androidTarget {
+		compilations.all {
+			kotlinOptions {
+				jvmTarget = "1.8"
+			}
+		}
+	}
+
```

Comment:
```
    listOf(
        iosX64(),
        iosArm64(),
        iosSimulatorArm64()
    ).forEach {
        it.binaries.framework {
            baseName = "shared"
            isStatic = true
        }
    }
```
Initial setup has above configuration for iOS but it results in 
`Build was configured to prefer settings repositories over project repositories but repository 'ivy' was added by build file 'shared/build.gradle.kts'`
To tackle that we can get rid of `repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)` from `settings.gradle.kts`. Aforementioned line is not present in project created from the template. 
So as a first step I removed the configuration for iOS to extract it to a separate PR.